core/dbt/task/init.py

```diff
@@ -244,6 +251,22 @@ def get_valid_project_name(self) -> str:
 
         return name
 
+    def create_new_project(self, project_name: str):
+        self.copy_starter_repo(project_name)
+        os.chdir(project_name)
+        with open("dbt_project.yml", "r+") as f:
```

Comment:
I suspect that the `cd` might actually be doing something downstream.

This code is kinda complicated, and I'd prefer to change as little as possible to avoid introducing an accidental regression.